test/integration/puppetlabs/services/master/environment_classes_int_test.clj

```diff
@@ -92,9 +94,47 @@
   [response]
   (-> response :body cheshire/parse-string))
 
-(deftest ^:integration environment-classes-integration-test
+(deftest ^:integration environment-classes-integration-cache-disabled-test
+  (testing "when environment classes cache is disabled for a class request"
+    (bootstrap/with-puppetserver-running-with-config
+     app
+     (-> {:jruby-puppet {:max-active-instances 1}}
+         (bootstrap/load-dev-config-with-overrides)
+         (ks/dissoc-in [:puppetserver
+                        :environment-class-cache-enabled]))
```

Comment:
@rlinehan Yeah, good points.  I should have explicitly set the value in the other tests in that namespace, too.  That was definitely an oversight.  Will include that in a later PR.

As for what the default setting in `dev/puppet-server.conf.sample` should be, I'm kind of mixed.  I generally have found it convenient to have the default value for a setting in the dev config be set to whatever "on" means for a feature that might be enabled/disabled, even if we've chosen to set the feature to "off" in shipping packages for backward compatibility.  But I suppose the argument could also be made that we should just have the dev config be as close as possible to the packaged default config - even if it means one would need to set things to "on" for development.

If you want to reset the value to the `dev` setting to "off" in a follow-up PR, I wouldn't have a problem with that.